### ISSUE TITLE
more training details of the TTS enhanced models

### ISSUE THREAD

**Issue by zjlww** [jasonppy/VoiceCraft#111]
Hi, thank you for open-sourcing your excellent work. ❤️

I would like to compare it with VoiceCraft as a baseline for my research. I have observed that you have released three TTS enhanced models. I am curious about the training datasets used for all these models. Can I utilize them to evaluate zero-shot TTS models?

**Comment by jasonppy** [comment]
Thanks!
830M TTS enhanced and 330M TTS enhanced (to be uploaded) are trained on gigaspeech + lightlight. I recommend using 830M TTS enhanced to evaluate.

**Comment by rlenain** [comment]
Hi @jasonppy -- I'm curious, if you can spare the details, how exactly did you train the TTS enhanced model compared to the base model? Is it a separate training script? Separate loss? Or simply separate data?

Thanks a lot.

**Comment by jasonppy** [comment]
> Hi @jasonppy -- I'm curious, if you can spare the details, how exactly did you train the TTS enhanced model compared to the base model? Is it a separate training script? Separate loss? Or simply separate data?
> 
> Thanks a lot.

The TTS enhanced model are trained without the first rearrange step introduced in the paper (i.e. no masking)

**Comment by rlenain** [comment]
Thanks !

**Comment by rlenain** [comment]
Sorry, actually there is something that I don't understand: is the TTS enhanced model trained from scratch as such, or simply finetuned with that specific objective (i.e. no masking) from the base 830m model? Is there a specific script / recipe that exists in the repo to train/finetune like you trained the TTS enhanced model?

Thanks a lot!

**Comment by jasonppy** [comment]
it's finetuned from the giga830M/giga330M that's trained with causal masking. Right now the scripts are not uploaded to the repo yet.

**Comment by Approximetal** [comment]
I tested the TTSEnhanced models, including the 330M and 830M. sometimes it repeats too long, or can't pronounce short words. Maybe we can set some rules to decide when to stop predicting, or add ASR post-processing to check if the pronunciation is correct.
![image](https://github.com/jasonppy/VoiceCraft/assets/25227045/a64f46f3-e10c-458d-bd9b-3700e04aacf4)
[test_sample.zip](https://github.com/user-attachments/files/15613595/test_sample.zip)
